packages/stryker-vue-mutator/test/unit/VueMutatorSpec.ts

```diff
@@ -0,0 +1,315 @@
+import { expect } from 'chai';
+import * as sinon from 'sinon';
+import { Config } from 'stryker-api/config';
+import { File } from 'stryker-api/core';
+import { Mutator, MutatorFactory, Mutant } from 'stryker-api/mutant';
+import VueMutator from '../../src/VueMutator';
+
+describe('VueMutator', () => {
+  let config: Config;
+  let sut: VueMutator;
+  let stubJavaScriptMutator: sinon.SinonStubbedInstance<Mutator>;
+  let stubTypeScriptMutator: sinon.SinonStubbedInstance<Mutator>;
+
+  beforeEach(() => {
+    config = new Config();
+    sut = new VueMutator(config);
+    stubJavaScriptMutator = sandbox.createStubInstance<Mutator>(VueMutator);
+    stubTypeScriptMutator = sandbox.createStubInstance<Mutator>(VueMutator);
+    sandbox.stub(MutatorFactory.instance(), 'create')
+      .withArgs('javascript', config).returns(stubJavaScriptMutator)
+      .withArgs('typescript', config).returns(stubTypeScriptMutator);
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  describe('with JavaScript', () => {
+    it('should throw an error if no JavaScript mutator is installed', () => {
+      sandbox.stub(MutatorFactory.instance(), 'knownNames').returns([]);
+      const vueFile = new File('Component.vue',
+        `<template>
+          <span id="msg">{{ message }}</span>
+        </template>
+        <script>
+          export default {
+            data () {
+              return {
+                message: 'hello!'
+              }
+            }
+          }
+        </script>`);
+
+      const files = [vueFile];
+
+      expect(() => sut.mutate(files)).throws(`The 'javascript' mutator is required to mutate a <script> block but it was not found. Please read the README of this package for information on configuration.`);
+    });
+
+    describe('when the first file is not a Vue file', () => {
```

Comment:
I think this describe is not helping with the explanation of what is tested in the `it`. Could we move the `it` to the top level and remove this `describe`?

Reply:
Fixed